docs/section11/Rule_11-10.md

```diff
@@ -0,0 +1,221 @@
+            - set has_fault to true: `has_fault = true`
+          - the setpoint temperature should be less than or equal to 180F: `if shw_eq.setpoint_temperature > 180:`
+            - add the following text to the note: "For Oil Storage Water heaters, the supply water temperature must be <= 180F": `rule_note += "For Oil Storage Water heaters, the supply water temperature must be <= 180F"`
+            - set has_fault to true: `has_fault = true`
+          - check whether the uniform_energy_factor is valid: `if uniform_energy_factor != NULL`
+            - check if the actual uniform_energy_factor is >= to the expected: `if uniform_energy_factor >= expected_UEF && !has_faults:`
+              - set rule_status to "PASS": `rule_status = "PASS"`
+          - otherwise, set rule status to UNDETERMINED and provide a note: `else: rule_status = "UNDETERMINED"; rule_note = "Uniform Energy Factor is required but not provided"`
+        - else the input capacity is >= 140,000 btu/hr: `else:`
+          - the capacity per volume should be < 4000 btu/hr/gallon: `if capacity_per_volume >= 4000:`
+            - add the following text to the note: "For Oil Storage heaters, with capacity > 140,000 btu/hr, the capacity per volume should be < 4000 btu/hr/gallon.": `rule_note += "For Oil Storage heaters, with capacity <= 140,000 btu/hr, the capacity per volume should be < 4000 btu/hr/gallon.  "`
+            - set has_fault to true: `has_fault = true`
+          - set the expected thermal efficiency to 0.8: `expected_thermal_efficiency = 0.8`
+          - make sure that the thermal efficiency is not NULL: `if thermal_efficiency != NULL:`
+            - check whether the uniform_energy_factor is valid: `if uniform_energy_factor != NULL`
+              - check if the actual uniform_energy_factor is >= to the expected: `if shw_eq.thermal_efficiency >= expected_thermal_efficiency && !has_faults:`
+                - set rule_status to "PASS": `rule_status = "PASS"`
+            - otherwise, set rule status to UNDETERMINED and provide a note: `else: rule_status = "UNDETERMINED"; rule_note = "Uniform Energy Factor is required but not provided"`
+          - otherwise set the rule status to UNDETERMINED and provide a note: `else: rule_status = "UNDETERMINED"; rule_note = "No thermal efficiency provided.  Expected thermal efficiency is: " + expected_thermal_efficiency`
+      
+   **Rule Assertion - Zone:**
+  - Case1: swh_type is INSTANTANEOUS - there is no baseline instantaneous water heater types: `if swh_type == "INSTANTANEOUS": UNDETERMINED; rule_note = "There baseline water heater is of an Instantaneous type.  There are no acceptable instantaneous baseline water heaters in Table G3.1.1-2."`
```

Comment:
suggest oil also be captured here or with another case with a similar note.

Reply:
suggest modifying note to ""There baseline water heater is of an Instantaneous type.  Only electric resistance and gas storage water heaters are applicable to the Appendix G baseline acccording to Table G3.1 #11 Baseline Building Performance column and Table G3.1.1-2 and therefore the efficiency of the modeled water heater was not assessed."`

Reply:
updated logic higher up (~line 55) to address this

Reply:
suggest modifying note to ""There baseline water heater is of an Instantaneous type. Only electric resistance and gas storage water heaters are applicable to the Appendix G baseline acccording to Table G3.1 https://github.com/pnnl/ruleset-checking-tool/issues/11 Baseline Building Performance column and Table G3.1.1-2. Consequently, the efficiency of the modeled water heater was not assessed."`